src/framework/application.js

```diff
@@ -212,27 +212,27 @@ pc.extend(pc, function () {
         this.loader.addHandler("textureatlas", new pc.TextureAtlasHandler(this.loader));
         this.loader.addHandler("sprite", new pc.SpriteHandler(this.assets, this.graphicsDevice));
 
-        var rigidbodysys = new pc.RigidBodyComponentSystem(this);
-        var collisionsys = new pc.CollisionComponentSystem(this);
-        var animationsys = new pc.AnimationComponentSystem(this);
-        var modelsys = new pc.ModelComponentSystem(this);
-        var camerasys = new pc.CameraComponentSystem(this);
-        var lightsys = new pc.LightComponentSystem(this);
+        new pc.RigidBodyComponentSystem(this);
+        new pc.CollisionComponentSystem(this);
+        new pc.AnimationComponentSystem(this);
+        new pc.ModelComponentSystem(this);
+        new pc.CameraComponentSystem(this);
+        new pc.LightComponentSystem(this);
         if (pc.script.legacy) {
             new pc.ScriptLegacyComponentSystem(this);
         } else {
             new pc.ScriptComponentSystem(this);
         }
-        var audiosourcesys = new pc.AudioSourceComponentSystem(this, this._audioManager);
-        var soundsys = new pc.SoundComponentSystem(this, this._audioManager);
-        var audiolistenersys = new pc.AudioListenerComponentSystem(this, this._audioManager);
-        var particlesystemsys = new pc.ParticleSystemComponentSystem(this);
-        var screensys = new pc.ScreenComponentSystem(this);
-        var elementsys = new pc.ElementComponentSystem(this);
-        var spritesys = new pc.SpriteComponentSystem(this);
+        new pc.AudioSourceComponentSystem(this, this._audioManager);
+        new pc.SoundComponentSystem(this, this._audioManager);
+        new pc.AudioListenerComponentSystem(this, this._audioManager);
+        new pc.ParticleSystemComponentSystem(this);
+        new pc.ScreenComponentSystem(this);
+        new pc.ElementComponentSystem(this);
+        new pc.SpriteComponentSystem(this);
         // var textsys = new pc.TextComponentSystem(this);
         // var imagesys = new pc.ImageComponentSystem(this);

```

Comment:
Please delete these two commented out lines as well, just to keep things tidy. 😄